### PR TITLE
feat: print cache location when no arg in deno info

### LINKED ISSUE #2749: Need a way to get the cache directory location
`deno info https://deno.land/welcome.ts` will tell me the location of the cached files associated with that URL, but there's currently no way to just quickly print the location of the cache directory. I think `deno info` without arguments should do that.

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes #2749

~~Before landing I'd like to show redirect chain for remote modules~~

### CODE REVIEW

**Comment by bartlomieju** [comment]
```
$ deno info
DENO_DIR location: "/Users/biwanczuk/Library/Caches/deno"
remote modules cache: "/Users/biwanczuk/Library/Caches/deno/deps"
TypeScript compiler cache: "/Users/biwanczuk/Library/Caches/deno/deps"
```

**Comment by nayeem-rahman** [comment]
> ```
> TypeScript compiler cache: "/Users/biwanczuk/Library/Caches/deno/deps"
> ```

Should be `.../gen`, shouldn't it?

File: cli/main.rs [hunk 1]
```diff
@@ -217,35 +133,129 @@ fn types_command() {
...
+  );
+  println!(
+    "{} {:?}",
+    ansi::bold("remote modules cache:".to_string()),
+    state.dir.deps_cache.location
+  );
+  println!(
+    "{} {:?}",
+    ansi::bold("TypeScript compiler cache:".to_string()),
+    state.dir.deps_cache.location
```
Comment (nayeem-rahman) [inline cli/main.rs]: Should be `state.dir.gen_cache.location`

Comment (bartlomieju) [reply cli/main.rs]: :+1: Thanks

File: cli/main.rs [hunk 2]
```diff
@@ -99,7 +98,62 @@ fn js_check(r: Result<(), ErrBox>) {
...
+  let state = worker.state;
+
+  println!(
+    "{} {:?}",
+    ansi::bold("DENO_DIR location:".to_string()),
+    state.dir.root
+  );
+  println!(
+    "{} {:?}",
+    ansi::bold("remote modules cache:".to_string()),
```
Comment (ry) [inline cli/main.rs]: Nit: the other lines start with capital letters - can you make “remote” capital too?

**Pull request merged**